x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -263,8 +263,10 @@ export const DETECTION_ENGINE_RULES_PREVIEW = `${DETECTION_ENGINE_RULES_URL}/pre
  * Internal detection engine routes
  */
 export const INTERNAL_DETECTION_ENGINE_URL = '/internal/detection_engine' as const;
-export const INTERNAL_DETECTION_ENGINE_RULE_STATUS_URL =
-  `${INTERNAL_DETECTION_ENGINE_URL}/rules/_find_status` as const;
+export const DETECTION_ENGINE_RULE_EXECUTION_EVENTS_URL =
+  `${INTERNAL_DETECTION_ENGINE_URL}/rules/{ruleId}/execution/events` as const;
+export const detectionEngineRuleExecutionEventsUrl = (ruleId: string) =>
+  `${INTERNAL_DETECTION_ENGINE_URL}/rules/${ruleId}/execution/events` as const;
```

Comment:
I'd argue that `common/constants.ts` is a bad file name and should be splitted into `urls.ts` and other stuff - instead of separating `DETECTION_ENGINE_RULE_EXECUTION_EVENTS_URL` and `detectionEngineRuleExecutionEventsUrl` - they are very closely related to each other.